frontend/src/constants.ts

```diff
@@ -146,17 +146,25 @@ export const DEFAULT_TRANSFORMS = Object.freeze([
     display_name: "Resize",
     object_name: "transforms.Resize",
     parameters: {
-      size: {
+      height: {
```

Comment:
Is src/constants.ts still being used? I thought it was a deprecated file, but if it is still being used, I suggest moving these constants to the other constants files (imageConstants, trainConstants, etc) for modularization & consistency purposes. Also I believe typescript's "as const" keyword provides better const typing than object.freeze

Reply:
@farisdurrani @karkir0003

Reply:
not too sure on this. faris might know better

Reply:
I'm not sure myself. It depends on what train models have been successfully refactored and whether those are using the new constants and not this one. It does look like some files are still using this constants file but that includes Home, Exercise, and ImageModels, all to be deprecated. Those files need to be deleted as well, otherwise removing constants.ts will cause a build fail. My recommendation is to leave this constants file where it is until we can properly deprecate and delete the old files